assets/shaders/octree/lightViewMap.frag.glsl

```diff
@@ -12,7 +12,7 @@ void main() {
         ((frag_position.xyz / frag_position.w) + vec3(1.0)) / 2.0,
         1.0
     );
-    uvec3 unnormalizedGlobalPosition = uvec3(ceil(normalizedGlobalPosition.xyz * float(voxelDimension)));
+    uvec3 unnormalizedGlobalPosition = uvec3(floor(normalizedGlobalPosition.xyz * float(1023)));
```

Comment:
Do we need `1023`? Or is `383` good enough? Also, we usually use the top value, for example `voxelDimension` instead of `voxelDimension - 1`.

I was using `float(voxelDimension) * 1.5` for example.